content/examples/enterprise-quickstart/docker-compose.yaml

```
@@ -0,0 +1,68 @@
+version: "3"
+networks:
+  main: {}
+services:
+  pomerium:
+    image: pomerium/pomerium:latest
+    volumes:
+      - ./config.yaml:/pomerium/config.yaml:ro
+    ports:
+      - 443:443
+    networks:
+      main:
+        aliases:
+        - authenticate.localhost.pomerium.io
+  pomerium_console:
+    networks:
+      main:
+    depends_on:
+      database:
+        condition: service_healthy
+      pomerium:
+        condition: service_started
+    image: docker.cloudsmith.io/pomerium/enterprise/pomerium-console:latest
+    expose:
+      - 8701
+      - 9090
+    environment:
+      - AUDIENCE=console.localhost.pomerium.io
+      - ADMINISTRATORS=admin@example.com
+      - DATABASE_ENCRYPTION_KEY=tXBj4gGDj45m8cW7ehhcy5lRuxsEeNl0X/nnsN5YJPw=
+      - DATABROKER_SERVICE_URL=pomerium://pomerium:5443
+      - SHARED_SECRET=mxGl062SqkrbQKvqG9R2jqHqxq1Oi1BNj2AAeZHNq7c=
+      - DATABASE_URL=postgresql://<user>:<password>@database/<database>?sslmode=disable
```

Maybe we should use postgres as a database name otherwise it requires extra steps for the user (create a database with <DB_name>). 
```suggestion
      - DATABASE_URL=postgresql://<user>:<password>@database/postgres?sslmode=disable
```